### ISSUE TITLE
Support for SK9822 LED strip

### ISSUE THREAD

**Issue by alanwangwang** [hyperion-project/hyperion#743]
i have some [SK9822 led strip](http://www.szledcolor.com/) ,now i want to use it decorate my house, 
Could it run on the Raspberry Pi? if it can work with Wifi too ?If there are no plans for this, do you think it is feasible?

**Comment by penfold42** [comment]
I don't think it will work just yet - the timing looks different to the existing led strips.

It shouldn't be hard to add support tho.

**Comment by penfold42** [comment]
My mistake - I read this as sk6822 - I'll look into 9822 support too

**Comment by alanwangwang** [comment]
@penfold42 ,ok  thank you very much

**Comment by penfold42** [comment]
Are you able to compile from source code ?

If so, I can add support to Hyperion.ng

**Comment by penfold42** [comment]
It's the same protocol as apa102

https://hyperion-project.org/threads/sk9822-same-as-apa102.342/

**Comment by witop** [comment]
The Sk9822 is similar with APA102 . learn more please link https://www.alibaba.com/product-detail/Sk9822ic-Full-color-addressable-led-strip_60626725449.html?spm=a2747.manage.list.7.yWj8fC   
Jessie
Jessie@witop-tech.com

**Comment by penfold42** [comment]
Can you be more specific on the actual differences ?

I can't learn more from a link selling your product.
Please don't link to sales sites - this is a place for technical contributions not product promotion.

**Comment by tpmodding** [comment]
deleted it...spam
